cli/configssh_test.go

```diff
@@ -230,44 +228,19 @@ func TestConfigSSH_FileWriteAndOptionsFlow(t *testing.T) {
 				}, "\n"),
 			},
 			wantConfig: wantConfig{
-				ssh: strings.Join([]string{
-					"Host myhost",
-					"	HostName myhost",
-					baseHeader,
-					"",
-				}, "\n"),
+				ssh: []string{
+					strings.Join([]string{
+						"Host myhost",
+						"	HostName myhost",
+					}, "\n"),
+					headerStart,
+					headerEnd,
+				},
 			},
 			matches: []match{
 				{match: "Continue?", write: "yes"},
 			},
 		},
-		{
-			name: "Section is not moved on re-run",
```

Comment:
This test case (and also `No prompt on no changes` below) is a bit tricky to support because it requires writing out a file that will exactly match what we generate. I can rework the test to make this possible if needed, though.

Reply:
Hmm, I think I'm fine losing these for now and I'll make a note to come back to it 😎